charts/ark-pentaho-ce/files/setenv.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+CATALINA_OPTS="-Xms550m -Xmx1024m  -Djava.rmi.server.hostname=localhost -XX:MaxPermSize=256m -Dsun.rmi.dgc.client.gcInterval=3600000 -Dsun.rmi.dgc.server.gcInterval=3600000"
+#export $CATALINA_OPTS
+#export CATALINA_OUT=/home/pentaho/log/pentaho/catalina.out
+#export CATALINA_TMPDIR=/home/pentaho/tmp/pentaho

```

Comment:
Is there a reason to keep commented out lines in the file?

Reply:
As Fabrice suggested tried removing this and it worked.